project_task_recurring_activity/models/recurring_activity.py

```diff
@@ -0,0 +1,115 @@
+from datetime import timedelta
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class RecurringActivity(models.Model):
+    _name = "recurring.activity"
+    _description = "Recurring activity"
+
+    project_task_id = fields.Many2one("project.task")
+    activity_type_id = fields.Many2one(
+        "mail.activity.type", string="Activity Type", ondelete="restrict"
+    )
+    user_id = fields.Many2one(
+        "res.users",
+        string="Assigned to",
+        index=True,
+        required=True,
+    )
+    summary = fields.Char(
+        string="Summary",
+    )
+    description = fields.Html(
+        string="Description",
+        sanitize_style=True,
+    )
+    days_after_task_creation_date = fields.Integer()
+    next_recurrence_date = fields.Date(
+        string="next_date", compute="_compute_next_recurrence_date", store=True
+    )
+
+    @api.depends("days_after_task_creation_date")
+    def _compute_next_recurrence_date(self):
```

Comment:
Hello, thank you for your contribution. Could you consider some improvements?

It would be great to have the possibility to choose the recurrence over:
- nb of days (you did it)
- weekly
- monthly
- yearly